### PR TITLE
refactor: returns root-hash if an identity is not found

### PR DESCRIPTION
**Pull request by shotonoff**
## Issue being fixed or feature implemented

The operation verifying an identity-id must not return an error if the identity doesn't exist

## What was done?

Added handling edge case when no proof for an identity is found

## How Has This Been Tested?

Unit-Test

## Breaking Changes

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/rs-drive/src/drive/verify/identity.rs [hunk 1]
```diff
@@ -342,6 +342,9 @@ impl Drive {
         } else {
             GroveDb::verify_query(proof, &path_query)?
         };
+        if proved_key_values.is_empty() {
+            return Ok((root_hash, None));
+        }
```
Comment (lklimek) [inline packages/rs-drive/src/drive/verify/identity.rs:347]: please fix error message in line 371 below:  `"expected one identity id"`

**Review by lklimek — APPROVED** [review]

**Review by QuantumExplorer — APPROVED** [review]
